accounts/forms.py

```diff
@@ -341,6 +341,7 @@ class Meta:
             ),
             "age": _("What is your age?"),
             "gender": _("What is your gender?"),
+            "gender_self_describe": _("Describe your gender"),
```

Comment:
@okaycj Can you check version control on this vs. current production? Just confused why this is showing up as an addition in this PR